crates/proof-of-sql/src/base/arrow/record_batch_conversion.rs

```diff
@@ -102,13 +102,15 @@ impl<C: Commitment> TableCommitment<C> {
 #[cfg(all(test, feature = "blitzar"))]
 mod tests {
     use super::*;
-    use crate::base::scalar::Curve25519Scalar;
+    use crate::base::{
+        commitment::naive_commitment::NaiveCommitment as RistrettoPoint,
```

Comment:
An alias works here, but I'd rather see `NaiveCommitment` replace `RistrettoPoint` in the tests. This is confusing.